### DISCUSSION TITLE
Launch an installed package with arguments

### DISCUSSION THREAD

**Discussion by MarkIngramUK** [microsoft/WindowsAppSDK#2000] [Ideas]
Unless I've missed something, there appears to be 2 APIs available for launching packaged applications:

Launch, but with no arguments
[Windows.ApplicationModel.Core.AppListEntry.LaunchAsync](https://docs.microsoft.com/en-us/uwp/api/windows.applicationmodel.core.applistentry.launchasync?view=winrt-22000)

Launch, with arguments
[IApplicationActivationManager.ActivateApplication](https://docs.microsoft.com/en-us/windows/win32/api/shobjidl_core/nf-shobjidl_core-iapplicationactivationmanager-activateapplication)

Will WindowsAppSDK be providing an equivalent `Microsoft.ApplicationModel.Core.AppListEntry` API? If so, can I suggest that an additional `LaunchAsyncWithArgs` function is added, so that we can stay within the WindowsAppSDK, without needing to dip into COM (`IApplicationActivationManager`)?

Related discussion - https://github.com/microsoft/WindowsAppSDK/issues/81#issuecomment-645661927

**Comment by BenJKuhn** [comment]
This is a great suggestion. I agree, having to use the COM API isn't always convenient. We have no plans for such an API right now, but I've moved this over to an issue / feature in the backlog so that we don't lose track of this input.
